core/domain/summary_services.py

```diff
@@ -131,3 +140,23 @@ def _generate_query(categories):
         })
 
     return results
+
+def get_featured_explorations():
+    """Returns a list of featured explorations."""
+    exp_summary_dict = []
+    exp_summary_dict = exp_services.get_non_private_exploration_summaries()
+    featured_exp_summary_dict = []
+    featured_exp_ids = []
+
+    for key in exp_summary_dict:
+        if exp_summary_dict[key].status == \
```

Comment:
Don't use backslashes. Do something like

         if exp_summary_dict[key].status == (
             rights_manager.ACTIVITY_STATUS_PUBLICIZED)

instead. (We avoid backslashes for the purposes of escaping newlines because it's not actually obvious what the backslash is escaping -- e.g., an extra space character after the backslash is hard to detect.)